tests/simulation/test_simulator.py

```diff
@@ -251,7 +251,7 @@ async def test_create_coins_with_invalid_amounts_raises(
 ) -> None:
     [[full_node_api], [[wallet_node, wallet_server]], _] = simulator_and_wallet
 
-    await wallet_server.start_client(PeerInfo("localhost", uint16(full_node_api.server._port)), None)
+    await wallet_server.start_client(PeerInfo("127.0.0.1", uint16(full_node_api.server._port)), None)
```

Comment:
is this change necessary because we no longer support hostnames or is it a matter of reducing test flakiness?

Reply:
The former, yes.